### PR TITLE
Fix money_only_cents for negative money

### PR DESCRIPTION
**Pull request by bertiecroll**
**What/Why**
When given negative money, `money_only_cents` view helper does not return the expected subunit value due to how modulo handles negative numbers.
For example: -125 % 100 = 75

**What was done**
To fix, I've updated the calculation to use the absolute value and added a new spec to cover the use case.

### CODE REVIEW

File: spec/helpers/action_view_extension_spec.rb [hunk 1]
```diff
@@ -105,6 +105,11 @@
       let(:monetizable_object){ false }
       it { is_expected.to eq "00" }
     end
+
+    context 'with a negative monetizable object' do
+      let(:monetizable_object){ Money.new(-125) }
```
Comment (sunny) [inline spec/helpers/action_view_extension_spec.rb:110]: Minor presentation fix to clarify the cents part here:

```suggestion
      let(:monetizable_object){ Money.new(-1_25) }
```

**Review by sunny — APPROVED** [review]
Looks good to me! 🔥

**Comment by bertiecroll** [comment]
Thanks for the quick review, I've added the suggested improvement 👍🏽

**Review by semmons99 — APPROVED** [review]

**Comment by semmons99** [comment]
heads up. we have failing tests here.

**Comment by semmons99** [comment]
ignore. I removed 2.6/2.7 from testing

**Pull request merged**